src/lib/api/documentRequests/documentRequest.js

```diff
@@ -111,6 +112,20 @@ class QueryBuilder {
     return this;
   }
 
+  withPhysicalItemProviderPid(physicalItemProviderPid, pid_type = null) {
```

Comment:
Passing null as a default parameter works properly but I'm not sure if it's a good practice so please give me feedback.